packages/contracts/EVENTS.md

```
@@ -87,6 +87,51 @@ Emitted when a yield source is removed.
 - **Topics**: `(REGISTRY, SOURCE_REMOVED, source_id: Symbol)`
 - **Data**: `{}`
 
+### VAL_ATT (value_attested)
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)